.github/workflows/build_release_artifacts.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Release artifacts
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  trigger_azdo_ci:
+    name: Download and Build artifacts
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/download-artifact@v3
+      with:
+        path: ${{ runner.temp }}\grpc-labview-artifacts

```

Comment:
Is it possible to create some env variables for these things so that we can re-use